lib/node_modules/@stdlib/math/base/special/chebyshev-t-polynomial/README.md

```diff
@@ -0,0 +1,154 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# chebyshevtpoly
+
+> [Chebyshev Polynomial][chebyshev-polynomial] of the first kind.
+
+<section class="intro">
+
+The [Chebyshev Polynomial][chebyshev-polynomial] of the first kind is defined as
+
+<!-- <equation class="equation" label="eq:chebyshev_polynomial" align="center" raw="T_n(x) = \cos( n \cos^{-1}(x) )" alt="Chebyshev Polynomial of the first kind."> -->
+
+```math
+T_n(x) = \cos( n \cos^{-1}(x) )
+```
+
+<!-- </equation> -->
+
+where `n` is the polynomial degree.
+
+The [Chebyshev Polynomial][chebyshev-polynomial] of the first kind is related to the Gaussian hypergeometric function via the following equation
```

Comment:
```suggestion
The [Chebyshev Polynomial][chebyshev-polynomial] of the first kind is related to the [Gaussian hypergeometric function][@stdlib/math/base/special/hyp2f1] via the following equation
```

You'll need to add the link defn below in the links section. Also double-check that I used the correct package name.